index.js

```diff
@@ -60,10 +60,12 @@ module.exports = function(config) {
         logger = makeLogger(name, name);
         context.logger = logger;
         var getConnection = require('./lib/api/get_connection')(context);
+        var events = require('./lib/api/events');
         context.foundation = {
             makeLogger: makeLogger,
             startWorkers: require('./lib/api/start_workers')(context),
-            getConnection: getConnection
+            getConnection: getConnection,
+            events: events
```

Comment:
What does this API represent? Simply calling it `events` is inconsistent with the other APIs and not particularly descriptive.